testing/jupyterlab/test-info-pages/no-tab-trap-initial-page.md

```diff
@@ -0,0 +1,38 @@
+# No Tab Trap on Initial Page
+
+- id: 1wlgh6
```

Comment:
where is this id created from?

Reply:
[random.org/strings](https://www.random.org/strings/?num=1&len=6&digits=on&loweralpha=on&format=html&rnd=new)

ACT [uses random ids](https://act-rules.github.io/pages/design/rule-design/) for their rules. That way the name of the test can change but it keeps the same id.